### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+This is the bug tracker for the Guests app. Find other components at https://github.com/owncloud/core/blob/master/.github/CONTRIBUTING.md#guidelines
+
+For fixing potential security issues please see https://owncloud.org/security/
+
+To make it possible for us to get your change reviewed and merged please fill out below information carefully.
+
+Please note that any kind of change first has to be submitted to the master branch which holds the next major version of ownCloud.
+
+We will carefully discuss if your change can or has to be backported to stable branches.
```
Comment (davitol) [inline .github/PULL_REQUEST_TEMPLATE.md]: No stable branches for Apps, just master and release branches

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<!--
+Thanks for reporting issues back to ownCloud! This is the issue tracker of ownCloud, if you have any support question please check out https://owncloud.org/support
+
+This is the bug tracker for the Guests app. Find other components at https://github.com/owncloud/guests/blob/master/.github/CONTRIBUTING.md#guidelines
+
+For reporting potential security issues please see https://owncloud.org/security/
+
+To make it possible for us to help you please fill out below information carefully.
+
+Before reporting any issues please make sure that you're using the latest available version for your major branch (e.g. 9.0.x), see https://owncloud.org/changelog/
```
Comment (davitol) [inline .github/issue_template.md]: 9.0. is no longer supported

File: .github/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+**PHP version:**
+
+**ownCloud version:** (see ownCloud admin page)
+
+**Updated from an older ownCloud or fresh install:**
+
+**Where did you install ownCloud from:**
+
+**Signing status (ownCloud 9.0 and above):**
```
Comment (davitol) [inline .github/issue_template.md]: 9.0. is no longer supported

**Comment by PVince81** [comment]
fixed and rebased

**Comment by PVince81** [comment]
@davitol please rereview

**Comment by davitol** [comment]
👍

**Pull request merged**